fury/ui.py

```diff
@@ -1420,6 +1429,30 @@ def _set_position(self, position):
         if self._background is not None:
             self._background.SetPosition(*self.actor.GetPosition())
 
+    @property
+    def position2(self):
+        """
+        Gets top-right corner position.
+
+        Returns
+        -------
+        (float, float, float)
+        """
+        return self.actor.GetPosition2()
+
+    @position2.setter
+    def _set_position2(self, position2=(100, 100, 0)):
+        """ Set text actor position2.
+
+        Parameters
+        ----------
+        position : (float, float)
+            The new position. (x, y) in pixels.
+        """
+        self.actor.SetPosition2(*position2)
+        if self._background is not None:
+            self._background.SetPosition2(*self.actor.GetPosition2())
```

Comment:
I do not think we should make this accessible to the user. The user already have size to update. I think you can remove this block (`_set_position2` and `position2`)

Reply:
Okay!